app/models/stake_account.rb

```diff
@@ -68,6 +70,7 @@ class StakeAccount < ApplicationRecord
   scope :filter_by_staker, ->(staker) { where('staker LIKE ?', "#{staker}%") }
   scope :filter_by_withdrawer, ->(withdrawer) { where('withdrawer LIKE ?', "#{withdrawer}%") }
   scope :active, ->() { where.not(active_stake: [0, nil]) }
+  scope :greater_than_one_sol, ->() { where("active_stake > ?", MINIMUM_STAKE) }
```

Comment:
I would change this scope to take minimum stake as a parameter that defaults to MINIMUM_STAKE to be more versatile (and call it `having_minimum_stake` or something similar)

Reply:
Sounds good. I did use the `with_minimum_stake`.